theano/sandbox/gpuarray/tests/test_blocksparse.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, print_function, division
+import numpy
+
+import theano
+from theano import tensor
+import theano.tests.unittest_tools as utt
+from theano.tensor.nnet.tests import test_blocksparse
+
+from .config import mode_with_gpu, test_ctx_name
+
+from ..type import gpuarray_shared_constructor
+from ..blocksparse import (GpuSparseBlockGemv,
+                           GpuSparseBlockOuter,
+                           gpu_sparse_block_gemv,
+                           gpu_sparse_block_outer)
+
+
+class BlockSparse_Gemv_and_Outer(test_blocksparse.BlockSparse_Gemv_and_Outer):
+    def setUp(self):
+        utt.seed_rng()
+        self.mode = mode_with_gpu.excluding('constant_folding')
+        self.gemv_op = gpu_sparse_block_gemv
+        self.outer_op = gpu_sparse_block_outer
+        self.gemv_class = GpuSparseBlockGemv
+        self.outer_class = GpuSparseBlockOuter
+
+    # This test is temporarily disabled since we disabled the output_merge
+    # and alpha_merge optimizations for blocksparse due to brokeness.
+    # Re-enable when those are re-added.
+    def Xtest_blocksparse_grad_merge(self):
```

Comment:
It was also disabled in the old backend for the same reason and I think it doesn't work on the CPU again for the same reason.